buf.gen.yaml

```diff
@@ -2,11 +2,11 @@ version: v1
 managed:
   enabled: true
 plugins:
-  - remote: buf.build/protocolbuffers/plugins/go:v1.28.1-1
+  - plugin: buf.build/protocolbuffers/go:v1.30.0
```

Comment:
Newer versions of the `buf` CLI use the `plugin` key for both local and remote plugin references.

I've also updated the plugin versions to the latest available upstream version and re-generated with `buf generate` version 1.17.0